### PR TITLE
clean up helpers

### PR DESCRIPTION
**Pull request by aschweigert**

### CODE REVIEW

File: inc/helpers.php [hunk 1]
```diff
@@ -217,9 +209,8 @@ function largo_youtube_iframe_from_url( $url, $echo = TRUE ) {
 	$output = '<iframe  src="//www.youtube.com/embed/' . largo_youtube_url_to_ID( $url ) . '" frameborder="0" allowfullscreen></iframe>';
 	if ( $echo ) {
 		echo $output;
-	} else {
-		return $output;
 	}
+	return $output;
```
Comment (rclations) [inline inc/helpers.php]: do you want to return `$output` right after you echo'd it?

Comment (aschweigert) [inline inc/helpers.php]: d'oh, one sec

**Comment by rclations** [comment]
👍 many thanks

File: inc/helpers.php [hunk 2]
```diff
@@ -278,32 +268,27 @@ function largo_make_slug( $string, $maxLength = 63 ) {
...
-function largo_get_current_url() {
-	$is_ssl = is_ssl();
-	if (!empty($is_ssl))
-		return "https://" . $_SERVER['SERVER_NAME'] . $_SERVER['REQUEST_URI'];
-	else
-		return "http://" .$_SERVER['SERVER_NAME'] . $_SERVER['REQUEST_URI'];
-}
+ function largo_get_current_url() {
+ 	$url = $_SERVER['SERVER_NAME'] . $_SERVER['REQUEST_URI'];
+ 	return ( ! empty( is_ssl() ) ) ? 'https://' . $url : 'http://' . $url;
```
Comment (benlk) [inline inc/helpers.php]: This line causes a fatal syntax error: https://travis-ci.org/INN/Largo/jobs/207500636#L220

Comment (rclations) [inline inc/helpers.php]: maybe the empty() should be removed, since is_ssl return boolean?

Comment (benlk) [inline inc/helpers.php]: This particular problem is only a problem in PHP < 5.5: https://stackoverflow.com/questions/1075534/cant-use-method-return-value-in-write-context#4328049

Do we want to keep PHP 5.3 support?

Comment (benlk) [inline inc/helpers.php]: Same commit on PHP 5.5 has a failed test that we've seen before: https://travis-ci.org/INN/Largo/jobs/207500651#L261

Largo issue for that test: https://github.com/INN/Largo/issues/1404

Comment (rclations) [inline inc/helpers.php]: WP supports PHP 5.2.4+ and MySQL 5.0+, so we should too

**Pull request merged**